client/src/main/scala/org/http4s/client/WebSocketClient.scala

```diff
@@ -0,0 +1,33 @@
+package org.http4s
+package client
+
+import fs2.{Sink, Stream}
+import java.net.SocketAddress
+import org.http4s.websocket.WebsocketBits.WebSocketFrame
+
+trait WebSocketClient[F[_]] {
```

Comment:
I imagine we'd add more methods here for higher level (i.e., ponging, closing, aggregating fragments).